### ISSUE TITLE
.NET Core installed, still asks to install .NET Core

### ISSUE THREAD

**Issue by Lyrix1337** [RumbleMike/ValorantStreamOverlay#129]
basically the title. Installed .NET Core, tried to launch, didnt work with the error, that it wants .NET Core installed.
Restarted PC.
Still happening

**Comment by iamMihirT** [comment]
Well I am also facing the same issue.

**Comment by monkaCode** [comment]
Are u sure that u had installed the .NET-Core x86 and not the .NET-Core x64? It's necessary for the program to run successfully.

**Comment by CarThax08** [comment]
Yes, you have to ensure you are downloading the x86 version, as the x64 version will not work.

**Comment by 25khattab** [comment]
I think its better to close the issue I had the same issue once I installed the x86 it's fixed

**Comment by 25khattab** [comment]
I think it's better to close the issue the solution is to download x86 .NET-Core I had the same problem

**Comment by bastianbeps** [comment]
u need install x86 version

**Comment by Whanos** [comment]
Can you close this @Lyrix1337 ?
